packages/ozone/src/mod-service/index.ts

```diff
@@ -720,13 +774,19 @@ export class ModerationService {
 
     if (subject) {
       const subjectInfo = getStatusIdentifierFromSubject(subject)
-      builder = builder
-        .where('moderation_subject_status.did', '=', subjectInfo.did)
-        .where((qb) =>
+      builder = builder.where(
+        'moderation_subject_status.did',
+        '=',
+        subjectInfo.did,
+      )
+
+      if (!includeAllUserRecords) {
+        builder = builder.where((qb) =>
           subjectInfo.recordPath
             ? qb.where('recordPath', '=', subjectInfo.recordPath)
             : qb.where('recordPath', '=', ''),
         )
+      }
```

Comment:
When `includeAllUserRecords` is `true` perhaps we want `qb.where('recordPath', '!=', '')` so that we only get records back.  (If it's an issue it's small, wouldn't block on this.)

Reply:
hmmm... I do see what you mean but the name of the param doesn't really imply that ONLY records will be returned so I think I would actually expect this to include both user's account entry and record entries to be returned by this.